### ISSUE TITLE
Får ikke alle værdier retur. 

### ISSUE THREAD

**Issue by JesperTange** [htvekov/iem3155_esphome#1]
Hello Htvekov

Do you have an explanation for why I'm getting NA (Not Available) back?

![image](https://github.com/htvekov/iem3155_esphome/assets/149792278/2ccb16a2-07b0-42ee-a83d-1dcf8f125796)

[15:17:52][D][sensor:094]: 'Voltage avg. All Phases': Sending state -nan V with 0 decimals of accuracy
[15:17:52][D][modbus_controller.sensor:025]: Sensor new state: -nan
[15:17:52][D][sensor:094]: 'L1 Active Power': Sending state -nan W with 0 decimals of accuracy
[15:17:52][D][modbus_controller.sensor:025]: Sensor new state: -nan
[15:17:52][D][sensor:094]: 'L2 Active Power': Sending state -nan W with 0 decimals of accuracy
[15:17:52][D][modbus_controller.sensor:025]: Sensor new state: -nan
[15:17:52][D][sensor:094]: 'L3 Active Power': Sending state -nan W with 0 decimals of accuracy

L1 Active Power | NA |  
L2 Active Power | NA |  
L3 Active Power | NA | 
Voltage avg. All Phases | NA |
 
The rest are values coming in.

**Comment by JesperTange** [comment]
Likewise, as planned, it has reset the values; now it does not count on the individual phases anymore?
![image](https://github.com/htvekov/iem3155_esphome/assets/149792278/5a78f5e0-8536-4470-bf9f-d6220f3d22f0)

**Comment by JesperTange** [comment]
[06:59:07][D][modbus_controller.sensor:025]: Sensor new state: 16023.22
[06:59:07][D][sensor:094]: 'Energy Import Total': Sending state 16023.21582 kWh with 3 decimals of accuracy
[06:59:07][D][modbus_controller.sensor:025]: Sensor new state: 0.00
[06:59:07][D][sensor:094]: 'Energy Export Total': Sending state 0.00000 kWh with 3 decimals of accuracy
[06:59:07][D][modbus_controller.sensor:025]: Sensor new state: 41.92
[06:59:07][D][sensor:094]: 'Daily Energy Import Total': Sending state 41.91500 kWh with 3 decimals of accuracy
[06:59:08][D][modbus_controller.sensor:025]: Sensor new state: 0.00
[06:59:08][D][sensor:094]: 'Daily Energy Import Phase 1': Sending state 0.00000 kWh with 3 decimals of accuracy
[06:59:08][D][modbus_controller.sensor:025]: Sensor new state: 0.00
[06:59:08][D][sensor:094]: 'Daily Energy Import Phase 2': Sending state 0.00000 kWh with 3 decimals of accuracy
[06:59:08][D][modbus_controller.sensor:025]: Sensor new state: 0.00
[06:59:08][D][sensor:094]: 'Daily Energy Import Phase 3': Sending state 0.00000 kWh with 3 decimals of accuracy

**Comment by JesperTange** [comment]
har selv løst det, den skal stå i 3PH4W

**Comment by htvekov** [comment]
> har selv løst det, den skal stå i 3PH4W

Exactly, glad you got it solved. Didn't actually expect anyone would use this quite expensive meter in other than full three phase wiring/configuration 😉

![image](https://github.com/htvekov/iem3155_esphome/assets/45469813/8dde6d37-7856-439e-8ce0-f6e0af480a51)

**Comment by JesperTange** [comment]
Sorry, I replied in Danish out of excitement, no, you may be right. Now I'll see if I can get the next 3 meters on the same esphome :)
